src/Components/ProgressBars.mdx

```diff
@@ -54,6 +58,7 @@ Uses a gradient to create a striped effect. Not available in IE8.
   <ProgressBar striped bsStyle="danger" now={80} />
 </Playground>
 
+<!--
```

Comment:
@kirbyyardley rather than commenting this out I would prefer to add your "DO NOT USE" label. If developers don't see it here, they may assume or know it is available and seek the docs from Bootstrap directly.